### PR TITLE
openapi test

### PR DESCRIPTION
**Pull request by iammuho**
If someone makes something in openapi, it's not easy to understand if there is an error. This workflow will also test the openapi file

Correct test: https://github.com/hellofresh/kangal/pull/30/checks?check_run_id=1192224039
Failed test: https://github.com/hellofresh/kangal/pull/30/checks?check_run_id=1192242077

### CODE REVIEW

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -41,6 +41,12 @@ jobs:
       - name: Run ShellCheck
         uses: ludeeus/action-shellcheck@master
 
+      - name: Run Swagger Test
```
Comment (vgarvardt) [inline .github/workflows/testing.yml:44]: maybe
```suggestion
      - name: Run OpenAPI Test
```

Comment (iammuho) [reply .github/workflows/testing.yml]: The worker is for swagger, that's why I wanted to call it swagger. But I've changed now

**Comment by s-radyuk** [comment]
+1

**Comment by vgarvardt** [comment]
👍

**Pull request merged**